docs/01 - Running & Configuring the Application/A - Running & Configuring the Application.md

```diff
@@ -18,31 +19,42 @@
+This section covers how to run the application from source code using the provided Kafka docker image configuration, 
+the Anchor Reference server and the secrets provided in the demo configuration files.
 
 1. Clone this repository: `git clone ssh://git@github.com:stellar/java-stellar-anchor-sdk.git`.
 2. Start a Kafka Queue service: `cd docs/resources/docker-examples/kafka && docker compose up`
 3. Start the Anchor Reference server: `./gradlew service-runner:bootRun --args=--anchor-reference-server`
-    - This uses the default configuration file at [`anchor-reference-server.yaml`], but you can use a custom configuration file by setting the `REFERENCE_SERVER_CONFIG_ENV` environment variable to the path of the configuration file, following the [Path to Yaml](#path-to-yaml) format.
+    - This uses the default configuration file at [`anchor-reference-server.yaml`], but you can use a custom 
+   configuration file by setting the `REFERENCE_SERVER_CONFIG_ENV` environment variable to the path of the 
+   configuration file, following the [Path to Yaml](#path-to-yaml) format.
 4. Start the Anchor Platform: `./gradlew service-runner:bootRun --args=--sep-server`
-    - This step requires you to set up a `STELLAR_ANCHOR_CONFIG`. You can test the application by using the default one with `export STELLAR_ANCHOR_CONFIG=file:<full-path-to-java-stellar-anchor-sdk>/platform/src/main/resources/example.anchor-config.yaml`,  
+    - This step requires you to set up a `STELLAR_ANCHOR_CONFIG`. You can test the application by using the default one 
+   with `export STELLAR_ANCHOR_CONFIG=file:<full-path-to-java-stellar-anchor-sdk>/platform/src/main/resources/example.anchor-config.yaml`,  
     - Eventually you'll need to set up your own configuration based on the `anchor-config-default-values.yaml`.
-    - You will need to export additional environment variables, depending on your configuration. An example of the variables you may need can be found in [`example.env`]
+    - You will need to export additional environment variables, depending on your configuration. An example of the 
+   variables you may need can be found in [`example.env`]
 5. Start the Stellar Observer: `./gradlew service-runner:bootRun --args=--stellar-observer`
     - This also needs the `STELLAR_ANCHOR_CONFIG` previously mentioned.
 
 ## Configuring the Project
 
 ### Config Files
 
-As mentioned previously, both the Anchor Platform and Anchor Reference server are configured using yaml and they have default configuration files:
+As mentioned previously, both the Anchor Platform and Anchor Reference server are configured using yaml and they have 
+default configuration files:
 
-- **Anchor Platform** default config file is located at [`anchor-config-defaults.yaml`].
+- **Anchor Platform** default config file is located at [`anchor-config-defaults-values.yaml`].
```

Comment:
Is it `anchor-config-defaults-values.yaml` or `anchor-config-default-values.yaml`?